CONTRIBUTING.md

```diff
@@ -225,6 +239,6 @@ Unfortunately, it is still a semi-manual process.
     *  Select the new version tag among existing and fill in a description text box with new release link and list of changes (see `CHANGELOG.md`)
     *  Press `Publish`.
 
-Upon commit [Travis CI][] builds `master`, `latest` branches and `v0.0.1` tag. The tag automatically initiates NPM publish process if the build is successful.
+Upon commit [Travis CI][] builds `main`, `latest` branches and `v0.0.1` tag. The tag automatically initiates NPM publish process if the build is successful.
```

Comment:
We no longer have Travis CI. Will we remove this line in #444? It would be nice to add a comment there.